src/query_results.js

```diff
@@ -73,99 +73,99 @@ module.exports = class QueryResult {
    * assemble the actual results.
    *
    * This is a recursive function, and it traverses the query graph as a tree, with
-   * every recursion passing its output queryNodeID and primaryID to the next call
+   * every recursion passing its output queryNodeID and primaryCurie to the next call
    * to use as a matching criteria for its input.
    *
    * This graphic helps to explain how this works:
    * https://github.com/biothings/BioThings_Explorer_TRAPI/issues/341#issuecomment-972140186
    *
    * The preresults returned from this method are not at all consolidated. They are
    * analogous to the collection of sets in the lower left of the graphic, which
-   * represents every valid combination of primaryIDs and kgEdgeIDs but excludes
+   * represents every valid combination of primaryCuries and recordHashs but excludes
    * invalid combinations like B-1-Z, which is a dead-end.
    *
    * NOTE: this currently only works for trees (no cycles). If we want to handle cycles,
    * we'll probably need to keep track of what's been visited.
    * But A.S. said we don't have to worry about cycles for now.
    *
    * @return {
-   *   inputQueryNodeID: string,
-   *   outputQueryNodeID: string,
-   *   inputPrimaryID: string,
-   *   outputPrimaryID: string,
-   *   queryEdgeID: string,
-   *   kgEdgeID: string,
+   *   inputQNodeID: string,
+   *   outputQNodeID: string,
+   *   inputPrimaryCurie: string,
+   *   outputPrimaryCurie: string,
+   *   qEdgeID: string,
+   *   recordHash: string,
    * }
    */
-  _getPreresults(
-    dataByEdge,
-    queryEdgeID,
+  _getUnconsolidatedResults(
```

Comment:
if we're using `ucResults`, should the method be `_getUCResults` to be consistent?

Reply:
My intent here was to use the `long form` of the vocab for the function so it's obvious what the short form stands for, however if you'd prefer more uniform naming we'd probably be better off returning to just the long form for clarity.

Reply:
My impression is that consistent, descriptive names are better for understandability and code maintenance, while shorter names are often more practical for one-off stats/analysis code. But the topic is somewhat subjective, so I'm not too invested in any particular option. If you have a strong preference for a particular option, I'm happy with it too.

Reply:
I don't have an opinion. I don't mind seeing a mix of long-form and short-form names, particularly when it's consistent (long-form being for functions)